### ISSUE TITLE
App fails to launch after updating to OS X 12.4

### ISSUE THREAD

**Issue by nevir** [PostgresApp/PostgresApp#672]
Was previously working great on OS X 12.3, but now on launch it immediately crashes. Interestingly, the menu item does launch successfully (was enabled on boot).

Stack:
```
2022-07-06 05:59:19.795 Postgres[5662:54456] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSConcreteTask terminationStatus]: task not launched'
*** First throw call stack:
(
	0   CoreFoundation                      0x00000001aa34d198 __exceptionPreprocess + 240
	1   libobjc.A.dylib                     0x00000001aa097e04 objc_exception_throw + 60
	2   Foundation                          0x00000001ab232150 -[NSConcreteTask dealloc] + 0
	3   Postgres                            0x0000000104d6d500 Postgres + 87296
	4   Postgres                            0x0000000104d67a6c Postgres + 64108
	5   Postgres                            0x0000000104d6937c Postgres + 70524
	6   Postgres                            0x0000000104d5a54c Postgres + 9548
	7   dyld                                0x0000000104e7108c start + 520
)
libc++abi: terminating with uncaught exception of type NSException
zsh: abort      /Applications/Postgres.app/Contents/MacOS/Postgres
```

Crash log:
[Postgres-2022-07-06-055730.ips.zip](https://github.com/PostgresApp/PostgresApp/files/9054925/Postgres-2022-07-06-055730.ips.zip)
 
Mitigation:
Blowing away `~/Library/Application Support/Postgres` allows the app to launch again

Interestingly, I can copy the data directory and run postgres against it directly with no errors (used the homebrew install of postgres). Note that the problematic db is set up for PG v13

**Comment by jakob** [comment]
If it happens right after launching the app, I assume the most likely issue is the new check for PostgreSQL binary versions that we added recently. Postgres.app tries to execute "postgres -V" for all of your configured servers to check the version. I assume that the crash happens when this call fails.

The problem is that the API we are using just crashes the app when an error occurs.

It could be that the problem happens when Rosetta isn't installed. Maybe the OS update removes Rosetta, and then Postgres.app crashes. Once you install Rosetta, the problem might go away.

We had a similar problem before, and it looks like the new check we added doesn't check for Rosetta. We'll have to fix that.
